docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-tensorflow.rst

```diff
@@ -0,0 +1,132 @@
+Quantization-aware Training (QAT) with tensorFlow 2
+===================================================
+
+Here, we provide the steps that are required to integrate QAT from NNCF into the training script written with
+PyTorch or TensorFlow 2:
```

Comment:
```suggestion
PyTorch or TensorFlow:
```

Reply:
Why PyTorch is here if this is dedicated document for TensorFlow?

Reply:
I missed this line, thank you for noticing!